encrypter.cpp

```diff
@@ -0,0 +1,212 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <ios>
+#include <random>
+#include <sstream>
+#include <utility>
+
+using Byte = char;
+
+//=====put_error=====//
+void put_error(const std::string& error_message)
+{
+  std::cout << error_message;
+  exit(1);
+}
+
+//=====shuffle=====//
+void shuffle(std::vector<Byte>& input)
+{
+  std::default_random_engine gen;
+
+  int i = 0;
+  for (std::vector<Byte>::iterator iter = input.begin(); iter < input.end(); ++i, ++iter)
```

Comment:
If you are using index access to vector it doesn't make sense to use iterator. just iterate using index from 0 to size()-1